mantle/cmd/kola/testiso.go

```diff
@@ -994,14 +994,15 @@ func testAsDisk(ctx context.Context, outdir string) (time.Duration, error) {
 //   - run coreos-installer on the mounted block device
 //   - unmount iscsi
 //
-// 5 - coreos-iscsi-vm.container start a coreos-assemble:
-//   - launch cosa qemuexec instructing it to boot from an iPXE script
+// 5 - coreos-iscsi-vm.container -> start a coreos-assembler conainer:
+//   - launch kola qemuexec instructing it to boot from an iPXE script
 //     wich in turns mount the iscsi target and load kernel
-//   - note the virtserial port device: we pass through the serial port that was created by kola for test completion
+//   - note the virtserial port device: we pass through the serial port
+//     that was created by kola for test completion
 //
 // 6 - /mnt/workdir-tmp/nested-ign.json contains an ignition config:
```

Comment:
Do we know why `/mnt/workdir-tmp` was used here?

That directory is significant in say a `cosa run` but I don't think it is here. If it's not important here (i.e. another directory could be used without issue) then maybe let's just put the files in `/var/` to avoid confusion?

Reply:
It's a leftover from when i was crafting this test manually and running it with cosa run